contributor-guides/translating-astro-docs.md

```diff
@@ -83,66 +106,55 @@ If you want to take your reviews to the next level, here are some more questions
+- Do the code samples' titles, syntax highlighting (like `js` or `astro`), and highlighted lines match the English version?
 - Are there any links that could be localized? (e.g. Wikipedia and MDN links)
 
-Our maintainer's team does not know all the languages available for translation! **When you think a PR is good to be merged** after your suggestions were addressed (if any), approve it through GitHub's "Review Changes" button or leave a "**LGTM!**" in the comments to let us know. (“LGTM” is an abbreviation of “Looks Good to Me” often used to approve pull requests.)
+Our maintainer's team does not know all the languages available for translation, and this is the valuable assistance that you are providing us! **When you think a PR is good to be merged** after your suggestions were addressed (if any), approve it through GitHub's "Review Changes" button or leave a "**LGTM!**" in the comments to let us know. (“LGTM” is an abbreviation of “Looks Good to Me”, often used to approve pull requests.)
 
 If you are not used to approving PRs or suggesting changes on GitHub, read about [how to review and suggest changes](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) on GitHub Docs.
 
-## Translation Process
-
-To start working on translating a part of [docs.astro.build](https://docs.astro.build/), the first step is to figure out where the content lives in this repo:
-
-1. Is the text in the navigation menu (left sidebar on desktop, hamburger menu on mobile)?
-
-    ➤ Go to `src/i18n/{language}/nav.ts`
+### Getting a review on your own PR
 
-2. Is the text in the search box or modal?
+After publishing your own i18n PR, make sure to share it in your language's `i18n-crew` thread, where other contributors will be able to notice it quickly and review it sooner. Do not spam repeatedly with requests to review. All contributors are voluntarily taking their time, and if a few weeks have passed without any reviews, Astro maintainers will gently nudge contributors and proceed with the review ourselves if necessary.
 
-    ➤ Go to `src/i18n/{language}/docsearch.ts`
+When you receive a review, do not worry if you see a lot of suggestions made correcting your work. It doesn't mean you did a bad job! There are lots of things to account for when translating, including choices made by previous translators which we must match for consistency. Each language crew makes its own decisions based on which words do or do not get translated, as well as the level of formality and whether certain phrases are acceptable, for example. And, everyone makes typos and mistakes, or and it usually takes other people to notice them! There is usually a lot of discussion in a translation PR because of our active, dedicated community who wants to see the best docs possible! 
```

Comment:
```suggestion
When you receive a review, do not worry if you see a lot of suggestions made correcting your work. It doesn't mean you did a bad job! There are lots of things to account for when translating, including choices made by previous translators which we must match for consistency. Each language crew makes its own decisions based on which words do or do not get translated, as well as the level of formality and whether certain phrases are acceptable, for example. And, everyone makes typos and mistakes, or and it usually takes other people to notice them! There is usually a lot of discussion in a translation PR because of our active, dedicated community that wants to see the best docs possible!
```